README.md

```diff
@@ -27,57 +28,59 @@ released copy of these modules.
 
 An example version could be `5.5.0.hibernate02` to contain Apache Lucene version `5.5`.
 
-## Download
-
-Released module ZIPs are available on [JBoss Nexus](https://repository.jboss.org/nexus/index.html#welcome)
```

Comment:
I think we should keep a section somewhere with a link to the published ZIPs. Maybe in "Non-Maven users"?

Reply:
funny, that was my plan as well :) Missed it, thanks.